app/operations/creates_data_requests.rb

```diff
@@ -0,0 +1,19 @@
+class CreatesDataRequests < ApplicationOperation
+  def call(obj, args)
+    data_request = DataRequest.new(
+      workflow_id: args[:workflow_id],
+      user_id: args[:user_id],
+      subgroup: args[:subgroup],
+      requested_data: args[:requested_data]
```

Comment:
Rails will verify that this value is acceptable for us, and the record will have validation errors if not (which will trigger the `rescue_from` added to `ApplicationController`.